### DISCUSSION TITLE
Does Keycloak have any well-known enterprise customers?

### DISCUSSION THREAD

**Discussion by Aleksandr-Filichkin** [keycloak/keycloak#20628] [Q&A]
Do we have a list of well-known companies that use Keycloak? I don't see any information about it.
It looks strange that there are no tech presentations at any conferences about the integration Keycloak, lessons learned and other topics about Keycloak.

**Comment by vmuzikar** [comment]
Hello, you can see a public list of adopters [here](https://github.com/keycloak/keycloak/blob/main/ADOPTERS.md).

**Comment by christian-2** [comment]
I do consider [CERN](https://home.cern/)'s [Single Sign-On service](https://auth.docs.cern.ch/), which is based on Keycloak, as an important reference.
